test/data/commands.json

```diff
@@ -1,4 +1,4 @@
 {
     "name": "sd-checkout-code",
-    "command": "echo Cloning github.com/screwdriver-cd/guide, on branch branchName && if [ ! -z $SCM_CLONE_TYPE ] && [ $SCM_CLONE_TYPE = ssh ]; then export SCM_URL=git@github.com:screwdriver-cd/guide; elif [ ! -z $SCM_USERNAME ] && [ ! -z $SCM_ACCESS_TOKEN ]; then export SCM_URL=https://$SCM_USERNAME:$SCM_ACCESS_TOKEN@github.com/screwdriver-cd/guide; else export SCM_URL=https://github.com/screwdriver-cd/guide; fi && sd-step exec core/git \"git clone --quiet --progress --branch branchName $SCM_URL $SD_SOURCE_DIR\" && sd-step exec core/git \"git reset --hard 12345\" && echo Reset to 12345 && echo Setting user name and user email && sd-step exec core/git \"git config user.name sd-buildbot\" && sd-step exec core/git \"git config user.email dev-null@screwdriver.cd\" && export GIT_URL=$SCM_URL.git && export GIT_BRANCH=origin/branchName"
+    "command": "echo Cloning github.com/screwdriver-cd/guide, on branch branchName && if git --version > /dev/null 2>&1; then export GIT_WRAPPER=\"\"; else export GIT_WRAPPER=\"sd-step exec core/git\"; fi && if [ ! -z $SCM_CLONE_TYPE ] && [ $SCM_CLONE_TYPE = ssh ]; then export SCM_URL=git@github.com:screwdriver-cd/guide; elif [ ! -z $SCM_USERNAME ] && [ ! -z $SCM_ACCESS_TOKEN ]; then export SCM_URL=https://$SCM_USERNAME:$SCM_ACCESS_TOKEN@github.com/screwdriver-cd/guide; else export SCM_URL=https://github.com/screwdriver-cd/guide; fi && $GIT_WRAPPER \"git clone --quiet --progress --branch branchName $SCM_URL $SD_SOURCE_DIR\" && $GIT_WRAPPER \"git reset --hard 12345\" && echo Reset to 12345 && echo Setting user name and user email && $GIT_WRAPPER \"git config user.name sd-buildbot\" && $GIT_WRAPPER \"git config user.email dev-null@screwdriver.cd\" && export GIT_URL=$SCM_URL.git && export GIT_BRANCH=origin/branchName"
```

Comment:
Ran this cmd in shell, doesn't work...
1. when `$GIT_WRAPPER=""`, running `"" anything` will get `"": not found`
2.  I tried not to set $GIT_WRAPPER, but it will still fail. I get `"git: not found`

I think we shouldn't even set the GIT_WRAPPER when there is local git and need to take care of the double quote around the actual `git` cmd for sd-step when using local git

Reply:
Thank you for your reviewing.
I fixed it to remove $GIT_WRAPPER and change double quote to single.